tests/conftest.py

```diff
@@ -22,9 +22,11 @@
 
 import pennylane as qml
 from semantic_version import Version
-from qiskit_ibm_provider import IBMProvider
+from qiskit_ibm_provider import IBMProvider, IBMProviderValueError
 from pennylane_qiskit import AerDevice, BasicAerDevice, BasicSimulatorDevice
 
+# pylint: disable=protected-access, unused-argument, redefined-outer-name
```

Comment:
`unused-argument` comes from the usage of `conftest` for building the device based on the `@pytest.mark.parametrize` arguments such as `shots`.